qualysapi/contrib.py

```diff
@@ -1,282 +0,0 @@
-# File for 3rd party contributions.
```

Comment:
I don't know if we are using this file and deleting this would break VulnWhisperer.